.env.example

```
@@ -0,0 +1,13 @@
+GITHUB_TOKEN=
+
+DB_PASSWORD=
+POSTGRES_HOST=
+POSTGRES_PORT=
+POSTGRES_DB=
+POSTGRES_USER=
+
+KEYCLOAK_CLIENT_SECRET=
+
+EUREKA_HOST=
+ZIPKIN_HOST=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)